public/ticketmanager.php

```diff
@@ -604,6 +763,7 @@
                 echo "<th>Game</th>";
                 echo "<th>Developer</th>";
                 echo "<th>Reporter</th>";
+                echo "<th>Resolver</th>";
```

Comment:
It looks like you forgot to actually populate the column for this header:
![image](https://user-images.githubusercontent.com/32680403/164873066-cccf59cf-63fd-420c-a249-be56e26a61f1.png)